DeepSpeech.ipynb

```diff
@@ -1274,10 +1255,11 @@
     "def run_inference(session, caption, data_set, results_params=None):\n",
     "    if results_params is None:\n",
     "        # Get the data_set specific graph end-points\n",
-    "        results_params = get_results_params(data_set)\n",
+    "        tower_decodings, tower_labels, _, tower_distances, tower_total_losses, _, _ = get_tower_results(data_set)\n",
```

Comment:
Right, but we don't - I would just wonder if this has any impact over a long run where we reconstruct the graph for every validation.